modules/wallet.go

```diff
@@ -187,10 +187,9 @@ type (
 		Drop()
 	}
 
-	// Wallet stores and manages siacoins and siafunds. The wallet file is
-	// encrypted using a user-specified password. Common addresses are all
-	// dervied from a single address seed.
-	Wallet interface {
+	// EncryptionManager can encrypt, lock, unlock, and indicate the current
+	// status of the EncryptionManager.
+	EncryptionManager interface {
```

Comment:
I'm not sure what the value is in splitting the Wallet interface up like this. Is an `EncryptionManager` or a `KeyManager` useful on its own?